org.yuzu_emu.yuzu.json

```diff
@@ -0,0 +1,212 @@
+            ],
+            "builddir": true,
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/catchorg/Catch2/archive/v2.13.4.tar.gz",
+                    "sha256": "e7eb70b3d0ac2ed7dcf14563ad808740c29e628edde99e973adad373a2b5e4df",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "project-id": 7680,
+                        "url-template": "https://github.com/catchorg/Catch2/archive/v$version.tar.gz"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "nlohmann-json",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/nlohmann/json/archive/v3.9.1.tar.gz",
+                    "sha256": "4cf0df69731494668bdd6460ed8cb269b68de9c19ad8c27abc24cd72605b2d5b",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "project-id": 141453,
+                        "url-template": "https://github.com/nlohmann/json/archive/v$version.tar.gz"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "boost",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./bootstrap.sh --with-libraries=context,container,system,headers",
+                "./b2 -j $FLATPAK_BUILDER_N_JOBS install --prefix=/app"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://dl.bintray.com/boostorg/release/1.75.0/source/boost_1_75_0.tar.bz2",
+                    "sha256": "953db31e016db7bb207f11432bef7df100516eeb746843fa0486a222e3fd49cb"
+                }
+            ]
+        },
+        {
+            "name": "yuzu",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
+                "-DENABLE_QT_TRANSLATION=ON",
+                "-DYUZU_ENABLE_COMPATIBILITY_REPORTING=ON",
+                "-DENABLE_COMPATIBILITY_LIST_DOWNLOAD=ON",
+                "-DYUZU_USE_QT_WEB_ENGINE=ON",
+                "-DDISPLAY_VERSION=mainline-0-549",
+                "-DBUILD_REPOSITORY=yuzu-emu/yuzu-mainline"
+            ],
+            "cleanup": [
+              "/share/man",
+              "/share/pixmaps"
+            ],
+            "post-install": [
+                "install -Dm644 ../org.yuzu_emu.yuzu.metainfo.xml /app/share/appdata/org.yuzu_emu.yuzu.metainfo.xml",
+                "desktop-file-install --dir=/app/share/applications ../dist/yuzu.desktop",
+                "echo 'StartupWMClass=yuzu' >> /app/share/applications/yuzu.desktop",
```

Comment:
desktop-file-edit can be used here, but probably worth a patch upstream.

Reply:
Agreed. I will be batching all the suggestions and fixes into one PR (hopefully) on the upstream.